dask/dataframe/_dtypes.py

```diff
@@ -18,6 +19,13 @@ def _(dtype):
     return pd.array(["a", pd.NA], dtype=dtype)
 
 
+if PANDAS_GT_150:
+
+    @make_array_nonempty.register(pd.ArrowDtype)
+    def _make_array_nonempty_pyarrow_dtype(dtype):
+        return dtype.empty(2)
```

Comment:
Cool - I wasn't aware of this `empty` trick until now. Do you think it could be a problem for all the data to be NA like this? Is there ever a good reason to use "non-null" sample data?

Reply:
Good question -- I'm not sure. I don't know how meaningful returning non-null data in `make_array_nonempty` is. My recollection (which could be wrong) is that the primary intention is for the returned object to have a non-zero shape with the correct dtype. From the test added here and me poking around locally, using `dtype.empty(2)` seems to work. Given this is a definite improvement on our current `main`, I'm inclined to give it a try. Happy to iterate though

Reply:
Just checking in, are you okay with the current approach, or would you prefer to see updates to this PR?

Reply:
Ah sorry - I forgot I responded to this. Given that the current state of things is that these extension dtypes don't work anyway, I have no problem with using `dtype.empty` for now.